lua-docs/content/reference/mutableshape.yml

```diff
@@ -13,15 +13,36 @@ constructors:
 
 properties:
 
+    - name: "Anchor"
+      type: "Number3"
+      description: |
+          Anchor is a point that you set as a way to specify what part of the [MutableShape] will be used when setting its [Position](/reference/shape#property-position).
+      samples:
+        - code: |
+            -- Anchor at the center of the shape:
+            myShape.Anchor = {0.5,0.5,0.5}
+
+    - name: "Depth"
+      type: "number"
+      description: "Returns [This]'s depth, measured in cubes."
+
+    - name: "Height"
+      type: "number"
+      description: "Returns [This]'s height, measured in cubes."
+
+    - name: "Width"
+      type: "number"
+      description: "Returns [This]'s width, measured in cubes."
+
 functions:
     - name: "AddBlock"
-      description: "Adds a [Block] to the [MutableShape]."
+      description: "Adds a [Block] to the [This]."
       arguments:
         - name: "block"
           type: "Block"
 
     - name: "GetBlock"
-      description: "Gets a [Block] from the [MutableShape]"
+      description: "Gets a [Block] from the [This]."
       arguments: 
```

Comment:
Getting with number3 would be nice

Reply:
I agree, let's make it work as is for now, but it's something we'll add for sure.